{{cookiecutter.repostory_name}}/bin/notify.py

```diff
@@ -0,0 +1,75 @@
+import argparse
+import os
+from datetime import datetime
+
+import requests
+
+msg_template = "*{timestamp}*\n{author}\n{changelog}"
+
+
+class SlackNotifier:
+    url = "https://slack.com/api/chat.postMessage"
+
+    def __init__(self, token, channel):
+        if not channel or not token:
+            raise Exception("Missing token or channel")
+
+        self.token = token
+        self.channel = channel
+
+    def notify(self, text):
+        print(f"Sending message to channel {self.channel}")
+        r = requests.post(
+            self.url,
+            data={
+                "text": text,
+                "channel": self.channel,
+                "token": self.token,
+            },
+        )
+        r.raise_for_status()
+
+
+def get_deployment_author():
+    return os.environ.get("sender") or "Deployed manually"
+
+
+def get_deployment_changelog():
+    # for now it's just message of last commit when deployed with github action
```

Comment:
it doesn't work if it's not a github action and it doesn't work at all if the commit is "merged branch xxx to yyy" because we still don't know what changes have been deployed